app/uk/gov/hmrc/apidefinition/controllers/controllers.scala

```diff
@@ -54,7 +54,7 @@ package object controllers {
 
   /** Used to improve the error messages that request.body.validate might return.
     */
-  def validationResult(errors: Seq[(JsPath, Seq[JsonValidationError])]): JsValue = {
+  def validationResult(errors: Seq[(JsPath, collection.Seq[JsonValidationError])]): JsValue = {
```

Comment:
I don't know what the way forward should be for these interactions with Play JSON but I have done things like this in other repos

`successful(UnprocessableEntity(validationResult(errors.map(error => (error._1, error._2.toSeq)).toSeq)))`